_security-plugin/index.md

```diff
@@ -8,20 +8,66 @@ redirect_from:
+
+### Authentication
+
+Authentication is used to validate the identity of users and works by verifying an end user’s credentials against a backend configuration. These credentials can be a simple name and password, a JSON web token, or a TLS certificate. Once the authentication domain extracts those credentials from a user’s request it can check their validity against the authentication backend.
+
+The backend used for validation can be OpenSearch's built-in internal user database – used for storing user and roles configurations and hashed passwords – or one of a wide range of industry-standard identification protocols such as Lightweight Directory Access Protocol (LDAP), Active Directory, Security Assertion Markup Language (SAML), and OpenID Connect. A common practice is to chain together more than one authentication method to create a more robust defense against unauthorized access. This might involve, for example, HTTP basic authentication followed by a backend configuration that specifies the LDAP protocol. See the [Backend configuration]({{site.url}}{{site.baseurl}}/security-plugin/configuration/configuration/) section to learn more about setting up the backend.
+
+### Access control
+
+Access control (or authorization) generally involves selectively assigning permissions to users that allow them to perform specific tasks, such as clearing the cache for a particular index or taking a snapshot of the cluster. However, rather than assign individual permissions directly to users, OpenSearch assigns these permissions to roles and then maps the roles to users. For more on setting up these relationships, see [Users and roles]({{site.url}}{{site.baseurl}}/security-plugin/access-control/users-roles/). Roles, therefore, define the actions that users can perform, including the data they can read, the cluster settings they can modify, the indexes to which they can write, and so on. Roles are reusable across multiple users, and users can have multiple roles.
+
+Another notable characteristic of access control in OpenSearch is the ability to assign user access through levels of increasing granularity. Fine grained access control (FGAC) means that a role can control permissions for users at not only the cluster level, but the index level, the document level, and even the field level. For example, a role may provide a user access to certain cluster-level permissions but at the same time prevent the user from accessing a given group of indexes. Likewise, that role may grant access to certain types of documents but not others, or even include access to specific fields within a document but exclude access to other sensitive fields. Field masking further extends the fine grained control of data access by providing options to mask certain types of data, such as a list of emails, which can still be aggregated but not made viewable to a role.
```

Comment:
Added Hyphens.
What do you think of: "... at not only the cluster level but also the index level..." since the preposition "at" precedes the phrase? (commutes to the second element of the phrase). Just trying to keep the language flowing.
No comma after "not others": I'm worried that running two of the same conjunctions together in one sentence (four ideas) without a pause makes for a mouthful. I don't want risk having the reader give up. If it reads fine to you, I'll delete it.
Yes, good idea (FGAC)